src/modules/Workspaces/WorkspacesSnapshotTool/SnapshotUtils.cpp

```diff
@@ -12,142 +12,20 @@
 #include <workspaces-common/WindowFilter.h>
 
 #include <WorkspacesLib/AppUtils.h>
+#include <tlhelp32.h>
+#include <winternl.h>
+#include <initguid.h>
+#include <PwaHelper.h>
+
+#pragma comment(lib, "ntdll.lib")
 
 namespace SnapshotUtils
 {
     namespace NonLocalizable
     {
         const std::wstring ApplicationFrameHost = L"ApplicationFrameHost.exe";
-    }
-
-    class WbemHelper
```

Comment:
As you removed `WbemHelper` class, please also remove related includes (`#include <comdef.h>` and `#include <Wbemidl.h>`) and dependency (`wbemuuid.lib` in project properties).